model_compression_toolkit/exporter/model_exporter/keras/int8_tflite_exporter.py

```diff
@@ -93,7 +93,7 @@ def _get_pointwise_layer_to_replace_dense(self, wrapped_layer: KerasQuantization
 
         # Create the point-wise layer
         pw_layer = Conv2D(**pw_cfg)
-        pw_layer.build(wrapped_layer.layer.input_shape)
+        pw_layer.build(wrapped_layer.input_shape)
```

Comment:
was this a bug or is something in the wrapper changed that I'm missing? (is there a new field 'input_shape' of the wrapper instead of the layer?)

Reply:
Now, the model builder uses the wrapped layers for building the model (instead of internal layers). The wrapped layers are used for inference thus they have the input_shape (instead of the internal layers).